flake.nix

```diff
@@ -92,26 +92,26 @@
                 #    (nixpkgs)/(niri's dev flake) uses `cargo` to build.
                 #    And this builds all crates in the same derivation.
                 #    That's why they don't override individual crates.
-                libspa-sys = attrs: {
+                libspa-sys = _attrs: {
                   nativeBuildInputs = [pkg-config rustPlatform.bindgenHook];
                   buildInputs = [pipewire];
                 };
 
-                libspa = attrs: {
+                libspa = _attrs: {
                   nativeBuildInputs = [pkg-config];
                   buildInputs = [pipewire];
                 };
 
-                pipewire-sys = attrs: {
+                pipewire-sys = _attrs: {
                   nativeBuildInputs = [pkg-config rustPlatform.bindgenHook];
                   buildInputs = [pipewire];
                 };
 
-                gobject-sys = attrs: {
+                gobject-sys = _attrs: {
                   nativeBuildInputs = [pkg-config glib];
                 };
 
-                gio-sys = attrs: {
+                gio-sys = _attrs: {
```

Comment:
I think these should be replaced by `const` rather than making their names underscore-prefixed.